src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1401,6 +1401,13 @@ private void SelectMatchingUpdatePackages(ShowUpdatePackageOptions updatePackage
                     packageItem.IsSelected = packagesToSelect.Contains(packageItem.Id, StringComparer.OrdinalIgnoreCase);
                 }
             }
+
+            var firstSelected = _packageList.PackageItems.FirstOrDefault(p => p.IsSelected);
+            if (firstSelected != null)
+            {
+                _packageList._list.SelectedItem = firstSelected;
```

Comment:
This seems out of scope... did we decide we want to select a package automatically? It is a perf hit that possibly won't be beneficial.
Either way, this could also be done in the loop as my other comment.

Reply:
It was a suggestion brought up when I demo'ed the feature to make a better user experience by selecting the top package from the list of selected packages. I think it makes for a better user experience so I've added it here as well as it's related to this change. What will be the performance impact of auto-selecting the top package? I know it will load the details, but is that happening on the UI thread? How significant is the performance concern in the way details are displayed?

Reply:
I'm not sure I agree that it's a better user experience. Can you refer me to that thread or notes from that?
Perf concern is what you described: whatever the cost is for selecting a package (believe this varies based on many factors).

Reply:
The feedback was provided during a demo meeting so there are no written notes or threads. Let's discuss this offline and I'll give more details.